TransIT.DAL/Models/Entities/SuppliersParts.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TransIT.DAL.Models.Entities
+{
+    public class SuppliersParts
```

Comment:
rename Entity to SuppliersPart ,
in the singular, without s

Reply:
Better rename to "SupplierPart"